manifest.md

```diff
@@ -175,6 +175,14 @@ If the `config.mediaType` is set to the empty value, the `artifactType` MUST be
 If the artifact does not need layers, a single layer SHOULD be included with a non-zero size.
 The suggested content for an unused `layers` array is the [empty descriptor](#guidance-for-an-empty-descriptor).
 
+The design of the artifact depends on what content is being packaged with the artifact.
+The decision tree below MAY be used to structure your artifact:
+
+1. Does the artifact consist of at least one file or blob? If yes, continue to 2. If no, specify the `artifactType`, and set the `config` and `layers` to their empty descriptor values.
+2. Does the artifact have additional JSON formatted metadata as configuration? If yes, continue to 3. If no, specify the `artifactType`, include the artifact in the `layers`, and set `config` to the empty descriptor value.
+3. Does the artifact require a different `artifactType` from the `config.mediaType`, to support multiple configuration formats or versions for the same artifact tooling? If yes, continue to 4. If no, specify the `config` with the metadata, include the artifact in the `layers`, but do not set the `artifactType`.
```

Comment:
This one has me confused; isn't the preference for artifacts to _always_ set `artifactType` except for cases of backwards compatibility (ie, where the format existed before `artifactType` was defined)?

Reply:
I'm not opposed to that, but it feels redundant.

Reply:
It is a little redundant, but given this is recommendations specifically for new artifact authors, don't we want to end with a world where _most_ non-image artifacts are setting `artifactType` consistently and the config media type fallback is rarely invoked? (Thinking especially about registry implementors here, where a consistent data model is more helpful than being slightly less redundant.)

Reply:
In the cases where existing artifacts are encountered (e.g. helm charts before we convince them to change their tooling), are we suggesting that registry implementers will eventually drop their support for those artifacts? Can they drop that support as soon as 1.1 is released?

Reply:
From today's call we consolidated the last two options, always recommending `artifactType`, but with a note at the end to clarify that existing artifacts do not do this.